.github/workflows/ci.yaml

```diff
@@ -54,7 +58,7 @@ jobs:
   static-typing:
     # In this job, we compare the output of mypy before and after the PR.
     if: github.event_name == 'pull_request'
-    runs-on: ubuntu-22.04  # minimimum 22.04 for python3-typeshed
+    runs-on: ubuntu-24.04 # minimimum 22.04 for python3-typeshed
```

Comment:
Not your bug if you would like to drop the comment or s/minimimum/minimum/, feel free :laughing: